_schemaV2.graphql

```diff
@@ -5405,6 +5413,40 @@ type GeneEdge {
   node: Gene
 }
 
+# A user-facing thematic grouping of Genes
+type GeneFamily {
+  featuredGeneLinks: [FeaturedGeneLink]
+  genes: [Gene]
+
+  # A globally unique ID.
+  id: ID!
+
+  # A type-specific ID likely used as a database ID.
+  internalID: ID!
+  name: String!
+
+  # A slug ID.
+  slug: ID!
+}
+
+# A connection to a list of items.
+type GeneFamilyConnection {
+  # A list of edges.
+  edges: [GeneFamilyEdge]
+
+  # Information to aid in pagination.
+  pageInfo: PageInfo!
+}
+
+# An edge in a connection.
+type GeneFamilyEdge {
+  # A cursor for use in pagination
+  cursor: String!
+
+  # The item at the end of the edge
+  node: GeneFamily
+}
+
```

Comment:
these seem unrelated but I'm pretty sure they should be in master already. They're definitely in the live staging schema so I think people have been committing without running the git hooks 🙈

Reply:
Eek, this looks like me. But I'm definitely not skipping any `husky` steps, at least not intentionally. Wondering how this happened 🤔

Reply:
💭 the pre-commit hook uses the staging env vars. Is the gene family stuff gated behind an env var? Because then I could imagine this happening by first adding the schema code in MP and PR-ing the changes *before* adding the env var to staging.

If not I'm happy to help debug!